docs/en/plugins.md

```diff
@@ -0,0 +1,3 @@
+# Plugins
+
```

Comment:
Maybe this document should be translated into English.

Reply:
Fixed.